applications/discussions/controllers.py

```diff
@@ -0,0 +1,83 @@
+from sqlalchemy.exc import IntegrityError
+import applications.discussions.dal as db
```

Comment:
All `Import abc as ABC` type imports should come before `from abc import xyz`

Remember : Import before From